etc/containerpilot.json5

```diff
@@ -0,0 +1,34 @@
+{
+  consul: "127.0.0.1:8500",
+  jobs: [
+    {
+      name: "preStart",
+      exec: ["/usr/local/bin/consul-manage", "preStart"],
+    },
+    {
+      name: "consul",
+      port: 8500,
+      exec: ["/bin/consul", "agent", "-server",
+             "-bootstrap-expect", "3",
```

Comment:
I've pushed this commit https://github.com/autopilotpattern/consul/pull/37/commits/4a9acb7232fa18771fd671396a6a91de6e9141d9 which should cover us. What do we think of that?